src/intents.cs

```diff
@@ -61,13 +61,17 @@ public enum INCancelWorkoutIntentResponseCode : nint {
 		FailureNoMatchingWorkout
 	}
 
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Unavailable (PlatformName.MacOSX)] // xtro mac !unknown-native-enum! INCarAirCirculationMode bound
```

Comment:
No need to remove them but, going forward, you don't need the comments in the source. The commit message and the changes to the xtro files makes it clear (if you're going thru git history).